lead-time/model/lead_time_model.py

```diff
@@ -0,0 +1,83 @@
+from fastapi import APIRouter, Header
+
+from pydantic import BaseModel
+from datetime import date
+
+router = APIRouter()
+
+
+class TimeRequest(BaseModel):
+    projectslug: str
+    from_date: str = None
+    to_date: str = None
+
+
+def get_lead_time_details(project_details,
```

Comment:
Move these methods to lead_time_service.py

Reply:
Sorry I think this is duplicate. Remove these methods from here. Keep only in service.

Reply:
Modified